models/Feedback.js

```diff
@@ -0,0 +1,30 @@
+// Mongoose model class for Feedback
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+//Schema for providing feedback after finishing a course.
+const feedbackSchema = new Schema({
```

Comment:
Consider adding a userId field to the feedback model. Might be useful in the future, and we could prevent users from reviewing the same course twice.

Reply:
We already though about that and discussed it with iara, she don't want users to be associated directly with a feedback entry, and she mentioned that users should only be able to submit feedback right after completing a course, so having a userid to for example edit the review in the future is not relevant